### PR TITLE
[#31041] Update search box,search box api style

### PR DESCRIPTION
**Pull request by bienvu**
Update search box,search box api style,position search on header

### CODE REVIEW

File: themes/agov/agov_whitlam/sass/components/search/_search.scss [hunk 1]
```diff
@@ -87,7 +76,44 @@
...
+
+  @include respond-to('medium') {
+    position: relative;
+    bottom: 0;
+  }
+
+  form {
+    @extend %search-form;
+  }
+}
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/search/_search.scss]: The positioning of the search form within the header should be controlled by the header component. Move this to a header__search element inside header.

File: themes/agov/agov_whitlam/sass/components/search/_search.scss [hunk 2]
```diff
@@ -87,7 +76,44 @@
...
+%header {
+  background: color(header-background);
+  position: relative;
+  margin-bottom: 80px;
+
+  @include respond-to('medium') {
+    margin-bottom: 0;
+  }
+}
+
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/search/_search.scss]: This should be in _header.

File: themes/agov/agov_whitlam/sass/components/search/search.hbs [hunk 3]
```diff
@@ -1,6 +1,6 @@
-<div class="wrap-demo">
+<div class="wrap-demo block-search">
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/search/search.hbs]: Please use component-based classes, not the funky Drupal classes.

**Comment by bienvu** [comment]
Need review.

**Comment by joannguyen** [comment]
please review and merge

**Comment by kimpepper** [comment]
Feedback has been addressed.

**Pull request merged**